src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/guilds/ProgressionRepositorySQLite.kt

```
@@ -214,22 +214,22 @@ class ProgressionRepositorySQLite(private val storage: Storage<Database>) : Prog
+    private fun parseUnlockedPerks(json: String): Set<net.lumalyte.lg.domain.values.PerkType> {
         if (json.isBlank() || json == "[]") return emptySet()
 
         return try {
             json.trim('[', ']')
                 .split(',')
                 .map { it.trim().trim('"') }
                 .filter { it.isNotEmpty() }
-                .map { net.lumalyte.lg.application.services.PerkType.valueOf(it) }
+                .map { net.lumalyte.lg.domain.values.PerkType.valueOf(it) }
                 .toSet()
         } catch (e: SQLException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)